src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -232,11 +537,141 @@ public void OnViewRendered()
+    /// </summary>
+    [Parameter]
+    public EventCallback<ResizeResult> OnResize { get; set; }
+
+    /// <summary>
+    ///     JS-Invokable method to return when the mouse wheel is scrolled.
+    /// </summary>
+    [JSInvokable]
+    public async Task OnJavascriptMouseWheel(double x, double y, double deltaY)
+    {
+        await OnMouseWheel.InvokeAsync(new MouseWheelResult(x, y, deltaY));
+    }
+    
+    /// <summary>
+    ///     Handler delegate for the view's Extent changing.
+    /// </summary>
+    [Parameter]
+    public EventCallback<MouseWheelResult> OnMouseWheel { get; set; }
+
+    /// <summary>
+    ///     JS-Invokable method to return when a layer view is created.
+    /// </summary>
+    /// <param name="layerView">
+    ///     The new <see cref="LayerView"/>
+    /// </param>
+    [JSInvokable]
+    public async Task OnJavascriptLayerViewCreate(LayerView layerView)
+    {
+        await OnLayerViewCreate.InvokeAsync(layerView);
+    }
+    
+    /// <summary>
+    ///     Fires after each layer in the map has a corresponding LayerView created and rendered in the view.
+    /// </summary>
+    [Parameter]
+    public EventCallback<LayerView> OnLayerViewCreate { get; set; }
+    
+    /// <summary>
+    ///     JS-Invokable method to return when a layer view is destroyed.
+    /// </summary>
+    /// <param name="layerView">
+    ///     The destroyed <see cref="LayerView"/>
+    /// </param>
+    [JSInvokable]
+    public async Task OnJavascriptLayerViewDestroy(LayerView layerView)
+    {
+        await OnLayerViewDestroy.InvokeAsync(layerView);
+    }
+    
+    /// <summary>
+    ///     Fires after a LayerView is destroyed and is no longer rendered in the view. This happens for example when a layer is removed from the map of the view.
+    /// </summary>
+    [Parameter]
+    public EventCallback<LayerView> OnLayerViewDestroy { get; set; }
+    
+    /// <summary>
+    ///     JS-Invokable method to return when a layer view is destroyed.
+    /// </summary>
+    /// <param name="layer">
+    ///     The <see cref="Layer"/> that failed to be added to the view.
+    /// </param>
+    /// <param name="exception">
+    ///     The Javascript error wrapped in a .NET Exception.
+    /// </param>
+    [JSInvokable]
+    public async Task OnJavascriptLayerViewCreateError(Layer layer, JavascriptException exception)
+    {
+        await OnLayerViewCreateError.InvokeAsync(new LayerViewCreateErrorResult(layer, exception));
+    }
+    
+    /// <summary>
+    ///     Fires after a LayerView is destroyed and is no longer rendered in the view. This happens for example when a layer is removed from the map of the view.
+    /// </summary>
+    [Parameter]
+    public EventCallback<LayerViewCreateErrorResult> OnLayerViewCreateError { get; set; }
+    
+    /// <summary>
+    ///     Set this parameter to limit the rate at which recurring events (pointer-move, drag, etc.) are returned.
```

Comment:
What events does this apply to?

Reply:
Good question. It applies to all "rapidly-firing" events: pointer-move, drag, mouse-wheel, resize, and extent-change. I will add this to the xml.